### ISSUE TITLE
Build for OS X 10.9 or create separate "compatibility" releases

### ISSUE THREAD

**Issue by jimeh** [rsms/fb-mac-messenger#214]
I'm currently running 10.9.x due to some issues with Yosemite (and haven't had time to try El Capitan yet).

Anyway, the updater didn't include any warning that the v0.1.5 now requires OSX 10.10.0 or later, rather than OSX 10.9.0 or later like v0.1.4 did. It might be worth including that info in the changelog :)

**Comment by semeano** [comment]
I had it updated automatically to the latest version and now I can't run in on Mavericks (10.9.5).
https://www.dropbox.com/s/zz6v25tcw7nvff6/Screenshot%202015-10-24%2011.16.14.png?dl=0

I don't know why it even updates to the latest version, if it isn't supported.

**Comment by mcescalante** [comment]
Yeah. I'm a little confused about this, and the website is unclear that support for 10.9.5 is lost. Clarification or something would be great.

**Comment by rsms** [comment]
3e0b75e9cf31472b4a9195fed2dd7a4eee635553 fixes an issue in the changelog, which should now prevent <0.1.5 running on OS X 10.9 from updating.

Leaving this issue open b/c I think the real fix should be to build for 10.9 again.

**Comment by rsms** [comment]
Related: 9c1f7ab94dd8f7cac71147289f39ba5399fdd377 and 21987b15ed173cb0664ad1579245f7b74cf29a01

**Comment by rsms** [comment]
From the readme file: "Are you running OS X 10.9? [Download version 0.1.4](http://fbmacmessenger.rsms.me/dist/Messenger-0.1.4.1440786644-f3436781b1e02ea1.zip)"
